enumerable.rb

```diff
@@ -0,0 +1,96 @@
+ 
+
+module Enumerable
+
+    def my_each
+        for x in 0...self.length
+            yield(self[x])
+            x += 1
+        end
+    end
+
+    def my_each_indexs
+        for x in 0...self.length
+            yield(self[x],x)
+            x += 1
+        end
+    end
+
+    def my_select
+        select_arr = []
+        self.my_each{|x| select_arr << x if yield(x)}
+        select_arr
+    end
+
+    def my_all?
+        initial = false
+        self.my_each {|elem| yield(elem) ? initial = true : initial = false }
+        initial
+    end
+    
+    def my_any?
+        self.my_each do |e|
+          return true if yield(e)
+        end
+        true
```

Comment:
This part of your code is not working. You are always returning true no matter what the block evaluates to, it needs a couple of changes there in order to provide the right output, please kindly give your method a good read, the mistake would be quite easy to find.

Reply:
Awesome!!!